tests/integration.rs

```diff
@@ -415,14 +415,25 @@ Caused by:
 
     #[test]
     #[cfg(feature = "extern-dependencies-tests")]
-    // Ignore until https://github.com/fermyon/spin-python-sdk/pull/65 is merged
     fn http_python_template_smoke_test() -> anyhow::Result<()> {
+        let prebuild = |env: &mut testing_framework::TestEnvironment<_>| {
+            let mut setup_venv = std::process::Command::new("python3");
+            setup_venv.args(["-m", "venv", "venv"]);
+            env.run_in(&mut setup_venv);
+            let mut activate_venv = std::process::Command::new("source");
```

Comment:
Does this actually work?  I.e. does it modify the environment of the current process?  I would think it only works inside a shell.

Reply:
Good point! I am not sure why the test does not pass in CI though even without it.